test/Microsoft.IdentityModel.Validators.Tests/AadIssuerValidatorTests.cs

```diff
@@ -1,24 +1,127 @@
+                    ExpectedResult = true,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_V1TemplateWithV2Issuer_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV1CommonAuthority,
+                    TokenIssuer = ValidatorConstants.AadIssuer,
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = false,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_V2TemplateWithV1Issuer_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV2CommonAuthority,
+                    TokenIssuer = ValidatorConstants.V1Issuer,
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = false,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_V2TemplateWithV2Issuer_Success")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV2CommonAuthority,
+                    TokenIssuer = ValidatorConstants.AadIssuer,
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = true,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_NullTemplate_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV1CommonAuthority,
+                    TokenIssuer = "",
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = false,
+                },
+
+                new AadIssuerValidatorTheoryData("ValidateIssuer_NullIssuer_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV1CommonAuthority,
+                    TokenIssuer = "",
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = false,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_NullTenantId_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadIssuerV1CommonAuthority,
+                    TokenIssuer = ValidatorConstants.AadIssuer,
+                    TenantIdClaim = "",
+                    ExpectedResult = false,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuer_PPETemplateWithV1Issuer_Failure")
+                {
+                    TemplatedIssuer = ValidatorConstants.AadInstancePPE + "/" + AadIssuerValidator.TenantIdTemplate,
+                    TokenIssuer =  ValidatorConstants.AadInstance + "/" + ValidatorConstants.TenantIdAsGuid,
+                    TenantIdClaim = ValidatorConstants.TenantIdAsGuid,
+                    ExpectedResult = false,
+                },
+                new AadIssuerValidatorTheoryData("ValidateIssuerSigningKey_V2SigningKeyIssuer_V1TokenIssuer_Failure")
```

Comment:
We should add a test case for 1.1 as well

https://login.microsoftonline.com/common/v1.1/.well-known/openid-configuration

Reply:
(I don't expect any issues)

Reply:
Good idea!

Reply:
Added tests in [this commit](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2648/commits/b2ff02ab4471d64cef6bd1548dcf28c00f5283ac)